server/api/apiRoutes.js

```diff
@@ -12,6 +12,9 @@ module.exports = function (app) {
   app.route('/add')
     .post(apiController.postAddMember);
 
+  app.route('/update')
```

Comment:
We're going to need /updateUser and /updateOrg routes so this should probably be renamed soon, but it's fine for now.